packages/pn-personafisica-webapp/src/components/Contacts/ContactsSummaryCards.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { AddCircle, Verified, WarningOutlined } from '@mui/icons-material';
+import { Card, Stack, Typography } from '@mui/material';
+
+import { AddressType, ChannelType, DigitalAddress } from '../../models/contacts';
+
+type Props = {
+  legalAddresses: Array<DigitalAddress>;
+  courtesyAddresses: Array<DigitalAddress>;
+};
+
+type ContactsSummaryCardProps = {
+  contacts: Array<DigitalAddress>;
+  isSercQEnabled?: boolean;
+  addressType?: AddressType;
+};
+
+const ContactsSummaryCard: React.FC<ContactsSummaryCardProps> = ({
+  contacts,
+  isSercQEnabled,
+  addressType,
+}) => {
+  const { t } = useTranslation('recapiti');
+  const hasAddress = contacts.length > 0;
+  const isCourtesyCard = addressType === AddressType.COURTESY;
+  const title = isCourtesyCard ? 'summary-card.courtesy-title' : 'summary-card.legal-title';
+
+  const getIcon = () => {
+    if (!hasAddress) {
+      if (isCourtesyCard && isSercQEnabled) {
+        return <WarningOutlined color="warning" data-testid="warningIcon" />;
+      }
+      return (
+        <AddCircle
+          color="primary"
+          data-testid="addIcon"
+          aria-label="summary-card.add-icon-arialabel"
+        />
+      );
+    }
+
+    return <Verified color="success" data-testid="verifiedIcon" />;
+  };
+
+  const getDescription = () => {
+    if (!hasAddress) {
+      return t('summary-card.no-address');
+    }
+
+    return contacts
+      .filter((contact) => contact.senderId === 'default')
```

Comment:
we don't need this filter, because we show the message independently from if there are special contacts or not